apps/common/src/main/resources/db/migration/canton-network/postgres/stable/V041__interface_views.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE interface_views_template
+(
+    acs_event_number            bigint not null REFERENCES acs_store_template (event_number) ON DELETE CASCADE,
+    interface_id_package_id     text   not null,
+    interface_id_qualified_name text   not null,
+
+    -- the DamlRecord corresponding to the interface's view (if available)
+    interface_view              jsonb,
+    -- if the view computation failed, this will contain the error for debugging purposes
+    view_compute_error          jsonb,
+    CONSTRAINT has_view_or_error CHECK ((interface_view is null and view_compute_error is not null) OR
+                                        (view_compute_error is null and interface_view is not null)),
+
+    -- the index will allow queryiing `WHERE interface_id_qualified_name = some_qualified_interface_id ORDER BY acs_event_number`.
+    -- At time of writing, interfaces cannot really be upgraded (therefore the qualified name would change on every upgrade).
+    -- However, this might change in the future, so we include the package_id in the primary key.
+    -- package_id is in the last position because it's not expected to be used in queries.
+    PRIMARY KEY (interface_id_qualified_name, acs_event_number, interface_id_package_id)
```

Comment:
Not sure if we'll ever need to reference it, but `V037__denormalize_update_history.sql` I even explicitly named primary keys. Better safe than sorry.